db/schema.rb

```diff
@@ -10,17 +10,17 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180810103847) do
+ActiveRecord::Schema.define(version: 20180813135333) do
 
-  create_table "ips", force: :cascade do |t|
+  create_table "ips", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
```

Comment:
What is this?

Reply:
Default Rails 5 behaviour as weird as it may be